src/rgw/rgw_sts.cc

```diff
@@ -0,0 +1,129 @@
+#include <iostream>
+#include <map>
+#include <sstream>
+#include <boost/uuid/uuid.hpp>
+#include <boost/uuid/uuid_generators.hpp>
+#include <boost/uuid/uuid_io.hpp>
+#include "common/config.h"
+#include "common/ceph_argparse.h"
+#include "common/debug.h"
+#include "common/ceph_time.h"
+#include "global/global_init.h"
+#include "include/assert.h"
+#include "include/str_list.h"
+#include "rgw_common.h"
+#include "rgw_b64.h"
+#include "rgw_sts.h"
+
+using namespace std;
+
+const string RGWSTS::key = "LswwdoUaIvS8ltyTt5jkRh4J50vUPVVHtR2YPi5kE";
+
+std::string timeval_to_string(struct timeval& tv)
+{
+  char buf[30];
+  struct tm result;
+  gmtime_r(&tv.tv_sec, &result);
+  strftime(buf,30,"%Y-%m-%dT%H:%M:%S", &result);
+  sprintf(buf + strlen(buf),".%dZ",(int)tv.tv_usec/1000);
+
+  return std::string(buf);
+}
+
+void RGWSTS::build_output() {
+  boost::uuids::random_generator generator;
+
+  boost::uuids::uuid uuid1 = generator();
+  boost::uuids::uuid uuid2 = generator();
+
+  access_key_id = to_string(uuid1);
+  secret_access_key = to_string(uuid2);
+
```

Comment:
Well, I changed the code to use whatever rgw uses to generate access_keys and secret_keys.